presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2153,6 +2153,20 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
         return result.build();
     }
 
+    @Override
+    public void commitPartitionForCreate(ConnectorSession session, ConnectorOutputTableHandle tableHandle, Collection<Slice> fragments)
```

Comment:
Also i think we may consider calling both methods simply `commitPartitions`. The ovearloading of the `ConnectorOutputTableHandle` / `ConnectorInsertTableHandle` should make it clean why there are two methods.